apps/www/src/lib/registry/new-york/ui/select/select-item.svelte

```diff
@@ -18,7 +18,7 @@
 	{disabled}
 	{label}
 	class={cn(
-		"relative flex w-full cursor-default select-none items-center rounded-sm py-1.5 pl-2 pr-8 text-sm outline-none data-[highlighted]:bg-accent data-[highlighted]:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
+		"relative flex w-full cursor-default select-none items-center rounded-sm py-1.5 pl-2 pr-8 text-sm outline-none hover:bg-accent data-[highlighted]:bg-accent hover:text-accent-foreground data-[highlighted]:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
```

Comment:
Same here, I don't think we need hover classes as highlighted should handle hover events and it can be weird if you leave your mouse on one item and then use your keyboard to navigate down, it will technically show two items as highlighted.

Reply:
> Same here, I don't think we need hover classes as highlighted should handle hover event

Yea that makes sense, but without hover mouseover it does not highlight on hover - but I am not sure if that is because melt-ui needs to be updated or something else?

>  It can be weird if you leave your mouse on one item and then use your keyboard to navigate down, it will technically show two items as highlighted.

I agree, assuming highlighted works in both cases, that's fine. But if we _do_ need to use `hover:` I don't think it would be a big deal as  I _assume_ (possibly incorrectly) someone using a keyboard to navigate would be less likely to also be using the mouse for navigation.